.github/workflows/check-links-cron.yaml

```
@@ -0,0 +1,55 @@
+name: Periodically Check Documentation Links

```

```suggestion
name: Check Documentation Links
```

It's only periodical when we trigger it with the cron job; but the workflow itself is not necessarily periodical, especially if we can trigger it from PRs touching the workflow file. I'd remove this word from the title